### ISSUE TITLE
3.3V Logic

### ISSUE THREAD

**Issue by ongun-kanat** [bvarner/pi485#10]
Can this design work with 3.3V logic but 5V power?

**Comment by bvarner** [comment]
In theory, it should!

You may need to adjust R8 and R9 to a lower value to supply enough current to the transistors that switch the signal LEDs on / off, but that should be it.

The BAT43 should still saturate around 3.3v, and the current applied to the trigger on the 555 should be sufficient.

If you give it a shot, I'd love to know!

**Comment by bvarner** [comment]
I poked around with the simulation I made of the timer circuit, and after switching the input clock to 3.3v, the timer side of things still triggers.

From what I've found on the max485 chips, 3.3v should be enough to drive the input pins. Just watch out for what voltage comes off the output pin -- it may be vcc. Your best bet would be an opto-isolator.

I'm pretty sure these really inexpensive boards have a lot more protection circuitry:
https://www.aliexpress.com/premium/RS485-TTL.html?d=y&blanktest=0&origin=y&SearchText=RS485+TTL&tc=ppc&isViewCP=y&catId=0

Things you probably should have -- like fuses, etc.
I even broke down and started using one, after having my own board hooked up for quite a few months.
